packer/plugin/plugin_test.go

```diff
@@ -78,33 +82,41 @@ func TestHelperProcess(*testing.T) {
 	case "invalid-rpc-address":
 		fmt.Println("lolinvalid")
 	case "mock":
-		fmt.Printf("%s|tcp|:1234\n", APIVersion)
+		fmt.Printf("%s|tcp|:1234\n", pluginsdk.APIVersion)
 		<-make(chan int)
 	case "post-processor":
-		server, err := Server()
+		server, err := pluginsdk.Server()
+		if err != nil {
+			log.Printf("[ERR] %s", err)
+			os.Exit(1)
+		}
+		err = server.RegisterPostProcessor(new(helperPostProcessor))
```

Comment:
Uh ! Unchecked error !

Reply:
Linter got mad at me. I doubt it's necessary, but it was easier to go with the flow.